control-plane/stor-port/src/types/v0/store/node.rs

```diff
@@ -152,7 +152,7 @@ pub struct NodeSpec {
     /// Endpoint of the io-engine instance (gRPC).
     endpoint: std::net::SocketAddr,
     /// Node labels.
-    labels: NodeLabels,
+    pub labels: NodeLabels,
```

Comment:
Use method with the same name...
```suggestion
    labels: NodeLabels,
```